client/src/components/Filter.css

```diff
@@ -18,27 +18,22 @@
 .filter-option {
 	color: lighten(var(--bg), 40%);
 	display: inline-block;
-	cursor: pointer;
+	transition: all 1s;
+	box-shadow: 0 0 15px 0px #2f2f2f;
 	font-weight: bold;
 	padding: 10px 20px;
 	border-radius: 25px;
+	cursor: pointer;
+	 margin-right: 10px;
 }
 
-.filter-section input[type="radio"]:checked + .filter-option {
+.filter-option:hover {
+	background: var(--bg); /* Change background color on hover */
 	color: white;
-	background: var(--bg);
-}
+  }
+
+
 
-.filter-option + .filter-section input[type="radio"] + .filter-option {
-	border-left: solid var(--borderWidth) var(--fg);
-}
```

Comment:
can you add something similar so that when I click on a category (filter like photos...) its colour changes for example.